impeller/playground/playground.h

```diff
@@ -30,6 +31,18 @@ enum class PlaygroundBackend {
   kVulkan,
 };
 
+inline RuntimeStageBackend PlaygroundBackendToRuntimeStageBackend(
```

Comment:
`constexpr` with an FML_UNREACHABLE at the end.

Reply:
Done